tests/ios/conftest.py

```diff
@@ -2,7 +2,6 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
It's too bad we can't share more of the code between the android and ios ones, rather than having to specify a lot of this twice.